zebra-consensus/src/transaction.rs

```diff
@@ -211,32 +240,33 @@ where
                                 .call((spend.rk, spend.spend_auth_sig, &sighash).into());
 
                             // Disable pending sighash check #1377
-                            //async_checks.push(rsp.boxed());
-
-                            // TODO: prepare public inputs for spends, then create
-                            // a groth16::Item and pass to self.spend
-
-                            // Queue the verification of the Groth16 spend proof
-                            // for each Spend description while adding the
-                            // resulting future to our collection of async
-                            // checks that (at a minimum) must pass for the
-                            // transaction to verify.
+                            // async_checks.push(rsp.boxed());
```

Comment:
Can we schedule #1377 in the next sprint, so this code actually gets tested?
cc @mpguerra